packages/core/package.json

```diff
@@ -25,6 +25,7 @@
   "author": "",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@gomomento/generated-types": "^0.61.0",
```

Comment:
we should not depend on anything grpc inside of this core package

Reply:
💯 to what matt said